example_sns_init.yaml

```diff
@@ -25,104 +31,307 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# The structure allows for mapping identities to human-readable names.
+# This is currently a placeholder field and must be left empty for now.
+Principals: []
 
+# Path to the SNS Project logo on the local filesystem. The path is relative
+# to the configuration file location, unless an absolute path is given.
+# Must have less than 341,334 characters, roughly 256 Kb. The only supported
+# format is PNG.
 logo: logo.png
+
+# URL to the dapp controlled by the SNS project.
+# Must be a string of max length = 512.
 url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
 
+# Configuration of the NNS proposal metadata and is shown only in
+# the NNS proposal.
+NnsProposal:
+    # The title of the NNS proposal. Must be a string of max 256 bytes.
+    title: "NNS Proposal to create an SNS named 'Rock Out'"
+
+    # The HTTPS address of additional content required to evaluate the NNS
+    # proposal. Must be a string of max 2,000 bytes.
+    url: "https://forum.dfinity.org"
+
+    # The description of the proposal. Must be a string of max 15,000 bytes.
```

Comment:
Are we measuring the length of strings in bytes or characters: a _single_ non-ASCII character can take _multiple_ bytes.

Reply:
I am not sure. 
I would however say that if 1) this will be caught at proposal submission in case it is wrong AND 2) it is more that just a few minutes of work to find that out, then I would not block progress of this PR to clarify this (nicer to be precise, but I think it is also a detail that projects can test)

Reply:
My findings:

1. We indeed measure proposal description size in bytes (that's good as we support Unicode).
2. The actual constraints are: min=10 bytes, max=2,000 bytes.

See https://sourcegraph.com/github.com/dfinity/ic@7b0a6b32519aada63cb05925d7459f209b7d04f5/-/blob/rs/sns/governance/src/types.rs?L1357

Edited.